examples/src/writeread.c

```diff
@@ -319,15 +343,20 @@ int main(int argc, char* argv[])
         printf("Aggregate Read BW is %.3lf MiB/s\n"
                "Effective Read BW is %.3lf MiB/s\n\n",
                aggr_read_bw, eff_read_bw);
+        printf("File laminate time is %.3lf s\n",
+               time_laminate.elapsed_sec_all);
```

Comment:
2 new lines get added to the output. The `writeread` CI tests will need to be updated:
Here:
https://github.com/LLNL/UnifyFS/blob/bc2579685d2d91ec4730fbadd011e4a1612349d8/t/ci/100-writeread-tests.sh#L71
and here:
https://github.com/LLNL/UnifyFS/blob/bc2579685d2d91ec4730fbadd011e4a1612349d8/t/ci/100-writeread-tests.sh#L89